project/plugins.sbt

```diff
@@ -41,6 +41,6 @@ addSbtPlugin("ch.epfl.scala"      % "sbt-scalajs-bundler"      % "0.15.0-0.6")
 
 // Used to find dependencies
 addSbtPlugin("net.virtual-void"   % "sbt-dependency-graph"     % "0.9.2")
-addSbtPlugin("com.timushev.sbt"   % "sbt-updates"              % "0.4.3")
+addSbtPlugin("com.timushev.sbt"   % "sbt-updates"              % "0.5.0")
```

Comment:
@tpolecat Do you think we still need this? Is it necessary considering scala-steward, I guess you want to check updates manually every now and then